docs/pages/04-Callbak/01-Onchange.md

```diff
@@ -0,0 +1,19 @@
+## Set onchange callback
+
+Set `onchange` after , when user operate (include mouse click、keyboard operate and so on.) change content , `onchange` will be auto emit.
+
+If you need change `onchange` delay time( when no operate XXX millisecond before,  `onchange` will be emit)， can access to `onchangeTimeout` set. More info see [history setting](../02-deal\ content/07-history.md).
```

Comment:
need to
‘被触发’ 用 ‘被执行’ 会不会好一点？executed
注意中文逗号不要出现
只见过从句省略主语的，所以主句还是别省主语
can access to `onchangeTimeout` set 这个用法没看太懂
参考：(it 做形式主语，指代后面的'通过 onchangeTimeout 配置'）
It can change the delay time of `onchange` to set the `onchangeTimeout` option, `onchange` will be executed automatically after XXX millisecond without any users' action. And more infomation about it, you can visit this[history setting](../02-deal\ content/07-history.md).